projects/plugins/crm/includes/ZeroBSCRM.Database.php

```diff
@@ -56,7 +56,7 @@
   $ZBSCRM_t['eventreminders']         = $wpdb->prefix . "zbs_event_reminders";
   $ZBSCRM_t['tax']                    = $wpdb->prefix . "zbs_tax_table";
   $ZBSCRM_t['security_log']           = $wpdb->prefix . "zbs_security_log";
-  $ZBSCRM_t['automation-workflows']  = $wpdb->prefix . 'zbs_workflows'; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase, Universal.WhiteSpace.PrecisionAlignment.Found
+  $ZBSCRM_t['automation-workflows']   = $wpdb->prefix . 'zbs_workflows'; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase, Universal.WhiteSpace.PrecisionAlignment.Found
```

Comment:
Weird... everything is aligned except this line. With 3 space is aligned, with 2 no...

Reply:
It's because all the current code above this line is not indented correctly, they all have 1 too many spaces in them 😄

That leaves 3 ways to handle it:
* Do it the correct